### ISSUE TITLE
Heigth of .cropImgWrapper is not set

### ISSUE THREAD

**Issue by 0x1ad2** [sconsult/croppic#169]
Hi Mat,

When I try to implement croppic in our CMS the height of .cropImgWrapper is not being set (it's set to 0) so It doesn't allow me to drag the image across the crop window.

Here are some GIF's to show what's going on

 **When uploading an image**
![When uploading an image](https://i.gyazo.com/872b93de21ee3a3a2b749c468a1617f1.gif)

 **When changing the height manually**
![When changing the height manually](https://i.gyazo.com/2007a5f81824c095a60e37e869947e42.gif)

**Comment by svilius** [comment]
Hello, 

Try not to set hight on "inspect element", set it directly in file

**Comment by 0x1ad2** [comment]
@svilius I do not set the height manually it's only for example. The height is set in the croppic.js file, take a look at the createModal method on line 324.

**Comment by 0x1ad2** [comment]
@sconsult any idea?

**Comment by svilius** [comment]
try to "turn off" other(your cms) js files, maybe they are conflicting?

**Comment by 0x1ad2** [comment]
@svilius yeah that's the problem I've tried to disable all the other Javascript and CSS files but without any results.

Also I'm not closing the windows it's looping...so I'm selecting a new file from my folder (you don't see that part because I didn't want to capture my whole pictures folder) and after selection the modal that allows you to crop opens.

**Comment by ahuggins** [comment]
The issue is the css the docs tell you to set up.

the html is this:
<div id="yourId"></div>

the docs css is this:
#cropContainerHeader {
	width: 200px;
	height: 150px;
	position:relative; /* or fixed or absolute */
}

but really, the '#cropContainerHeader' should actually match the id in the html, so for the docs example the css should be:
#yourId {
	width: 200px;
	height: 150px;
	position:relative; /* or fixed or absolute */
}

I thought I was missing something, but as soon as I figured that out, it started working perfectly. That's kind of an annoying issue in the docs. I think a comment should be added to at least hint that this should match the id of the html...might seem obvious to the plugin author, but copy and pasting the docs code should work. I would recommend updating the docs to reflect that these html/css ids should match.

**Comment by sconsult** [comment]
Hi @ahuggins 

I will take your positive critisism on our documentation and have it updated to make that aspect clearer.
Cheers

Mat

**Comment by aleangelico** [comment]
Great @ahuggins, you are absolutely right!!!! I had a similar problem (the crop area was showing fine in Chrome but not in FireFox) and your solutions solved it !
Thanks mate

**Comment by metallurgical** [comment]
Really nice mate!
